uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_leave-calendar.scss

```diff
@@ -68,8 +53,23 @@ td.chr_leave-calendar__name {
   }
 }
 
-//Public Holiday, Weekend and Non Working day needs to be specified before other styles
-//to give other styles more priority
+.chr_leave-calendar__item--disabled {
+  cursor: not-allowed;
+  opacity: 0.5;
+}
+
+// Public Holiday, Weekend and Non Working day needs to be specified before other styles
+// to give other styles more priority
+.chr_leave-calendar__item--type--late {
+  background-color: #eca67e;
```

Comment:
Be kind and don't make me increase the scope of this PR even a little bit please